packages/app/src/components/popup/settings-popover.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useCallback } from 'react';
+import { Box, Text, BoxProps, Flex } from '@stacks/ui';
+import useOnClickOutside from 'use-onclickoutside';
+import { useWallet } from '@common/hooks/use-wallet';
+import { useDrawers } from '@common/hooks/use-drawers';
+import { useAnalytics } from '@common/hooks/use-analytics';
+import { ScreenPaths } from '@store/onboarding/types';
+import { AccountStep } from '@store/recoil/drawers';
+import { Divider } from '@components/divider';
+import { USERNAMES_ENABLED } from '@common/constants';
+
+const SettingsItem: React.FC<BoxProps> = ({ onClick, children, ...props }) => (
+  <Box
+    {...props}
+    width="100%"
+    p="base-tight"
+    color="ink.1000"
+    _hover={{ backgroundColor: 'ink.150' }}
+    cursor="pointer"
+    onClick={e => {
+      onClick?.(e);
+    }}
+  >
+    <Text textStyle="body.small" display="block">
+      {children}
+    </Text>
+  </Box>
+);
+
+export const SettingsPopover: React.FC = () => {
```

Comment:
Is popover a common term?

Reply:
yeah -> https://material-ui.com/components/popover/